src/NServiceBus.Persistence.AzureStorage/obsoletes.cs

```diff
@@ -0,0 +1,75 @@
+﻿#pragma warning disable 1591
+
+namespace NServiceBus
+{
+    using System;
+
+    [ObsoleteEx(Message = "Azure Storage Queues and Azure Service Bus supports timeouts natively and does not require timeout persistence.",
```

Comment:
I decided to include Azure Service Bus into the obsoletion message to give this a broader scope from the perspective of azure customers.